.github/workflows/update_combined_metadata.yaml

```diff
@@ -13,26 +13,45 @@ jobs:
     - name: Checkout curatedMetagenomicData
       uses: actions/checkout@v2
 
-    - name: Install remotes and curatedMetagenomicDataCuration
+    - name: Install curatedMetagenomicDataCuration
       run: |
         install.packages("remotes")
         remotes::install_github("waldronlab/omicidxClientR")
         remotes::install_github("waldronlab/curatedMetagenomicDataCuration")
       shell: Rscript {0}
 
     - name: Generate combined_metadata.rda
+      id: generate_metadata
       run: |
         library("dplyr")
         library("data.table")
         library("curatedMetagenomicDataCuration")
-        combined_metadata <- curatedMetagenomicDataCuration:::makeCombinedMetadata()
+        message(paste("****", Sys.time(), "Make combined metadata ****"))
+        combined_metadata <-
+            curatedMetagenomicDataCuration:::makeCombinedMetadata()
         path <- paste(Sys.getenv("GITHUB_WORKSPACE"),
                       "data/combined_metadata.rda",
                       sep = "/")
         save(combined_metadata, file = path, compression_level = 9)
+        message(paste("\n****", Sys.time(), "Check curation ****"))
+        status <- "passing"
+        tryCatch(curatedMetagenomicDataCuration::checkCuration(combined_metadata),
+             warning = function(w) {
```

Comment:
This is some nice coding @jwokaty, it doesn't look like code from someone who just started learning R a couple months ago!!